.gitignore

```diff
@@ -41,6 +41,9 @@ Thumbs.db
 # Next.js
 .next
 
+# NX Cache
+/.nx/cache
```

Comment:
This should already be covered by `.nx/` below

Reply:
Done.